cmake/BuildTools.cmake

```diff
@@ -246,10 +246,18 @@ function (add_fetch_gbench
 
     # detect if the "DISABLED" flag has been passed to this test
     set(is_disabled FALSE)
+    set(include_gtest FALSE)
```

Comment:
is this related/relevant?

Reply:
Needed in order to build the benchmarks annoyingly